### PR TITLE
changed logo

### PR DESCRIPTION
**Pull request by sedrubal**
as discussed in #331 I updated the logo. More precisely I made `app.svg` as logo to be displayed in the apps dropdown.

Additionally I made a new move image and removed the `Drag'N'Drop on the chart title to re-order` text, wich looks bad and does not give the user more information as he can clearly see...

### CODE REVIEW

**Comment by hypery2k** [comment]
Can one of the admins verify this patch?

**Comment by mckaygerhard** [comment]
i reviewed the pacht, looks good @sedrubal .. but the unused translation maybe are due your version of oc, in my oc the text are displayed in the charts page... that commit must be reverted

**Comment by sedrubal** [comment]
Hi, thanks for reviewing. I removed the whole `div` in the charts page so it wont be displayed anymore. Or did I miss to remove it anywhere?

**Comment by mckaygerhard** [comment]
not, i noted in the commit u removed.. i refers why removed the message?, i mean why removed the div causing removal of the message? due the new icon that makes an idea of movement?

**Comment by sedrubal** [comment]
I do not understand. I removed the message, because I think there is no need for it, because the icon (the old and the new one) are enough and the text breaks the owncloud design... If you want to keep it I'll undo it...

File: storagecharts2/src/main/php/templates/tpl.charts.php [hunk 1]
```diff
@@ -42,11 +42,6 @@
 }
 ?>
 
-<div id="storagecharts2" data-sc-sort="<?php  print($scSort)?>" data-sc-size="<?php  print($huSize)?>" data-sc-size-hus="<?php  print($huSizeHus)?>">
```
Comment (mckaygerhard) [inline storagecharts2/src/main/php/templates/tpl.charts.php]: i understand the icon its sefl-explanatory.. but this line cares me.. i test just now and feedback in few minutes... if are good the sorting click div's then i'll commit this pull request.

Comment (sedrubal) [inline storagecharts2/src/main/php/templates/tpl.charts.php]: what''s the problem? is this `div` needed?

Comment (hypery2k) [inline storagecharts2/src/main/php/templates/tpl.charts.php]: yes this div is needed!

**Comment by sedrubal** [comment]
Oops ok. I have undone deleting the div. But I don't see any difference...

**Comment by sedrubal** [comment]
thank you :D

**Comment by mckaygerhard** [comment]
thanks for the logo..

**Pull request merged**